### PR TITLE
feat(oracle): add multi-arch support

### PR DESCRIPTION
**Pull request by DmitriyLewen**
## Description
Add arches for OracleLinux.
Use same structure as for RockyLinux - https://github.com/aquasecurity/trivy-db/pull/321.

db size before - 559 Mb
db size after - 570 Mb

example for different version for different arches:
изображение

### CODE REVIEW

**Comment by DmitriyLewen** [comment]
@knqyf263 i updated this PR.
Take a look, when you have time.

File: pkg/vulnsrc/oracle-oval/oracle-oval.go [hunk 1]
```diff
@@ -204,22 +211,57 @@ func (vs *VulnSrc) commit(tx *bolt.Tx, ovals []OracleOVAL) error {
...
+	fixedVersionsByArch := make(map[string][]string)
+	for _, entry := range advisories.Entries {
+		arch := entry.Arches[0] // Before merging `arches` always contains only 1 arch
+		fixedVersions := []string{
+			entry.FixedVersion,
+		}
+		if savedVersion, ok := fixedVersionsByArch[arch]; ok {
+			fixedVersions = append(fixedVersions, savedVersion...)
+		}
+		fixedVersionsByArch[arch] = fixedVersions
```
[the remark this answers is not in the document]
Comment (knqyf263) [reply pkg/vulnsrc/oracle-oval/oracle-oval.go:227]: Actually, I have another idea. I'll try it.

[the remark this answers is not in the document]
Comment (knqyf263) [reply pkg/vulnsrc/oracle-oval/oracle-oval.go]: Updated. @DmitriyLewen Can you take a look?
https://github.com/aquasecurity/trivy-db/pull/331/commits/d65b44101b2968885d53863cb8d639cd83a5aced

[the remark this answers is not in the document]
Comment (DmitriyLewen) [reply pkg/vulnsrc/oracle-oval/oracle-oval.go]: > Can't we write as below?

hmm... that's an easier way 👍 I didn't think about that. Thanks, I'll use it somewhere else later 🙂

> Updated. @DmitriyLewen Can you take a look?

Looks good 👍

[the remark this answers is not in the document]
Comment (knqyf263) [reply pkg/vulnsrc/oracle-oval/oracle-oval.go]: Thanks for checking. I'll review https://github.com/aquasecurity/trivy/pull/4809 and then approve this PR.

**Review by knqyf263 — APPROVED** [review]
https://github.com/aquasecurity/trivy/pull/4809 looks good to me. Approved.

**Comment by DmitriyLewen** [comment]
@knqyf263 I merged this PR and updated https://github.com/aquasecurity/trivy/pull/4809 (go.mod and go.sum files).
Take a look, please

**Pull request merged**